File: Code/App/v2/USG/Scripts/App/Models/USGExaminationViewModel.js
function USGExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.PatientInfo = new PatientsModel(data.PatientInfo);
    this.USGComment = new
        USGExaminationCommentViewModel(data.USGComment);
    this.examinationDescriptionUSG = ko.observable();

};

File: Code/App/separateDB/USG/ViewModels/USGExaminationViewModel.cs
using RepositoryClasses.Models;

namespace USG.ViewModels
{
    public class USGExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public USGExaminationCommentViewModel USGComment { get; set; }
        public int PatientDieseaseId { get; set; }
    }
}